File: Bravotaskpy/task/chaptwo/investmentreturn.py
def investment_rate(percentage):
    INVESTMENT_PERCENTAGE = 7
    return INVESTMENT_PERCENTAGE
print(investment_rate(7))

ANNUAL_RATE = 7
amount_invested = 1000
percentage_rate = 7/100

def investment_return_after_10_years(ten_years):
    ten_years_return = (1 + percentage_rate) ** ten_years
    money_after_10_years = amount_invested * ten_years_return
    money_after_10_years = round(money_after_10_years, 2)
    return f"${money_after_10_years}"
print(investment_return_after_10_years(10))


def investment_return_after_20_years(twenty_years):
    twenty_years_return = (1 + percentage_rate) ** twenty_years
    money_after_20_years = amount_invested * twenty_years_return
    money_after_20_years = round(money_after_20_years, 2)
    return f"${money_after_20_years}"
print(investment_return_after_20_years(20))

def investment_return_after_30_years(thirty_years):
    thirty_years_return = (1 + percentage_rate) ** thirty_years
    money_after_30_years = amount_invested * thirty_years_return
    money_after_30_years = round(money_after_30_years, 2)
    return f"${money_after_30_years}"

print(investment_return_after_30_years(30))

File: Bravotaskjv/src/chaptwo/InvestmentReturn.java
package chaptwo;

import static java.lang.Math.round;

public class InvestmentReturn {

    public static void main(String[] args) {
        int tenYears = 10;
        double get10Years = after10Years(tenYears);

        int twentyYears = 20;
        double get20Years = after10Years(twentyYears);

        int thirtyYears = 30;
        double get30Years = after10Years(thirtyYears);

        System.out.println("$"+get10Years);
        System.out.println("$"+get20Years);
        System.out.println("$"+get30Years);

    }



        int ANNUAL_RATE = 7;
        static double amount_invested = 1000;
        static double percentage_rate = (double) 7 / 100;

    public static double after10Years(int tenYears) {
        double  money_Year10 = 0;
        double ten_years_return = Math.pow((1 + percentage_rate), tenYears);
        double money_after_10_years = amount_invested * ten_years_return;
        money_Year10 = Math.round(money_after_10_years * 100.0) / 100.0;

        return money_Year10;
    }

    public static double after20Years(int twentyYears) {
        double money_Year20 = 0;
        double twenty_Years_Return = Math.pow((1 + percentage_rate), twentyYears);
        double money_After_10_Years = amount_invested * twenty_Years_Return;
        money_Year20 = Math.round(money_After_10_Years * 100.0) / 100.0;

        return money_Year20;
    }

    public static double after30Years(int thirtyYears) {
        double money_Year30 = 0;
        double thirty_Years_Return = Math.pow((1 + percentage_rate), thirtyYears);
        double money_After_10_Years = amount_invested * thirty_Years_Return;
        money_Year30 = Math.round(money_After_10_Years * 100.0) / 100.0;

        return money_Year30;
    }


}
